lib/StaticAnalyzer/Checkers/SpecialReturnValueChecker.cpp

```diff
@@ -0,0 +1,136 @@
+//==- SpecialReturnValueChecker.cpp ------------------------------*- C++ -*-==//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
please describe the configuration dependencies of this checker: the statistics file path and how that file is used